hack/make-rules/verify.sh

```diff
@@ -48,8 +48,6 @@ fi
 # Exclude vendor checks in certain cases, if they're running in a separate job.
 if [[ ${EXCLUDE_GODEP:-} =~ ^[yY]$ ]]; then
   EXCLUDED_PATTERNS+=(
-    "verify-godeps.sh"             # runs in separate godeps job
-    "verify-godeps-licenses.sh"    # runs in separate godeps job
```

Comment:
these are still needed as long as https://github.com/kubernetes/test-infra/issues/12045 is unresolved. the only reason the verify-godeps CI job is green on this PR is because of https://github.com/kubernetes/kubernetes/issues/76094

Reply:
You're right, this shouldn't be merged until https://github.com/kubernetes/test-infra/pull/12104 is.